src/compiler/utilities.ts

```diff
@@ -215,8 +215,37 @@ namespace ts {
 
     // Gets the nearest enclosing block scope container that has the provided node
     // as a descendant, that is not the provided node.
-    export function getEnclosingBlockScopeContainer(node: Node): Node {
-        let current = node.parent;
+    export function getEnclosingBlockScopeContainer(node: Node): Node;
+
+    /**
+      * Gets the nearest enclosing block scope container that has the provided node

```

Comment:
Perhaps this could be worded:

"Gets the smallest block scope strictly containing 'node'."

What do you think?